ci.ts

```diff
@@ -54,10 +55,6 @@ export async function getArgs(
   // Detect and use the deno.json file automatically
   const options = await getDenoConfig();
 
-  if (options) {
-    denoArgs.push(`--config=deno.json`);
-  }
```

Comment:
Deno detects this since [v1.18.0](https://github.com/denoland/deno/releases/tag/v1.18.0), more than half a year ago. I don't think anyone actually using lume will be still using Deno that old.

Reply:
Fair enough